src/query/execute_schema.c

```diff
@@ -3984,6 +3985,7 @@ do_create_partition (PARSER_CONTEXT * parser, PT_NODE * alter, SM_PARTITION_ALTE
 
   reuse_oid = (smclass->flags & SM_CLASSFLAG_REUSE_OID) ? true : false;
   tde_algo = (TDE_ALGORITHM) smclass->tde_algorithm;
+  replication_opt = sm_is_replication_class (pinfo->root_op);
```

Comment:
NIT:

SM_CLASS를 이미 획득했고, reuse_oid나 tde_algo 획득 방식과 일치

```suggestion
  replication_opt = (smclass->flags & SM_CLASSFLAG_DATA_REPLICATION_OFF) ? false : true;
```

Reply:
@hornetmj 반영했습니다.